GameServer/packets/Client/168/DetailDisplayHandler.cs

```diff
@@ -2129,8 +2131,16 @@ public static string DelveStyle(GameClient clt, int id)
 				//.Value("Skill",43)
 				if (style.GrowthRate>0)
 					dw.AddKeyValuePair("OpeningDamage",style.GrowthRate*100);
-				//.Value("SpecialValue", GetSpecialValue(style),GetSpecialValue(style)!=0)
-				//.Value("FollowupStyle",style.DelveFollowUpStyles,!string.IsNullOrEmpty(style.DelveFollowUpStyles))
+                //.Value("SpecialValue", GetSpecialValue(style),GetSpecialValue(style)!=0)
+                //.Value("FollowupStyle",style.DelveFollowUpStyles,!string.IsNullOrEmpty(style.DelveFollowUpStyles))
+                if (style.Procs != null && style.Procs.Count > 0)
+                {
+                    foreach (Tuple<Spell, int, int> proc in style.Procs)

```

Comment:
Can we really have multiple tooltips in the proc?
Let's imagine we have multiple `style.Procs` where `style.Procs.Count > 1`.
In this case you will loop over the different procs and in the end the `MiniDelveWriter` which ensures the unicity of the key-values will erase all the previous procs and you will end up with only the last proc in the list being written in the tooltip.
Here, suggest to take the first or the last proc in the list: `style.Procs.FirstOrDefault()`.